front-end/src/app/task.service.ts

```diff
@@ -0,0 +1,26 @@
+import { jsDocComment } from '@angular/compiler';
+import { Injectable } from '@angular/core';
+import { Task } from 'src/model/Task';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TaskService {
+  readonly url = 'http://localhost:3000/tasks';
+  db: Task[] = [];
+
+  async getTasks(): Promise<Task[]> {
+    /*const data = await fetch(this.url);
+    return (await data.json()) ?? []; TODO: HA LESZ BACKEND AKKOR EZT HASZNÁLJUK!*/
+    return new Promise((resolve, reject) => {
+      resolve(this.db);
+    });
+  }
+
+  submitTask(id: number, title: string, comment: string) {
+    if (title != '') {
+      this.db.push({ id: id, title: title, comment: comment });
+    } else alert("Title can't be empty!");
```

Comment:
a materialnak van tök látványos megoldása ([snackbar](https://material.angular.io/components/snack-bar/overview)) ehelyett, az alert helyett kérlek használd ezt

Reply:
másik ideillő dolog, hogy ha az else ág fut le, akkor is reset-eled a formot. Ez user szemszögből baj, mert pl ha a user 400 karaktert beírt title nélkül a commenthez, submitol egyet véletlen, akkor kap egy hibaüzenetet, és azt kezdheti majd előről bepötyögni a 400 karaktert

Reply:
jött módosítás, de még mindig nem az igazi: jelenleg nem fut le az ellenőrzés, enged csak kommentet beírva taskokat felvinni. Ezt majd még átbeszéljük bővebben